nameko/messaging.py

```diff
@@ -389,6 +355,14 @@ def connection(self):
         )
         return Connection(self.amqp_uri, heartbeat=heartbeat)
 
+    def handle_message(self, provider, body, message):
+        ident = "{}.handle_message[{}]".format(
```

Comment:
Does it matter? The `identifier` is only used to help identify thread when they're killed: https://github.com/nameko/nameko/blob/master/nameko/containers.py#L470